src/components/v2/V2Project/V2FundingCycleSection/ReservedTokensSplitsCard.tsx

```diff
@@ -124,18 +139,19 @@ export default function ReservedTokensSplitsCard({
               {tokenAddress && tokenAddress !== constants.AddressZero ? (
                 <div style={smallHeaderStyle}>
                   {tokensTextSingular} contract address:{' '}
-                  <FormattedAddress address={tokenAddress} />
+                  <FormattedAddress address={tokenAddress} truncateTo={3} />
                 </div>
               ) : null}
```

Comment:
Thoughts on removing this entirely? This is already available in the **Tokens** section.

Reply:
Agreed. Unnecessary in this spot. Removed